Client/Frontend/Browser/TabManagerStore.swift

```diff
@@ -73,33 +74,37 @@ class TabManagerStore: FeatureFlagsProtocol {
     
     func removeScreenshot(forTab tab: Tab?) {
         if let tab = tab, let screenshotUUID = tab.screenshotUUID {
-            imageStore?.removeImage(screenshotUUID.uuidString)
+            _ = imageStore?.removeImage(screenshotUUID.uuidString)
         }
     }
 
-    // Async write of the tab state. In most cases, code doesn't care about performing an operation
-    // after this completes. Deferred completion is called always, regardless of Data.write return value.
-    // Write failures (i.e. due to read locks) are considered inconsequential, as preserveTabs will be called frequently.
-    @discardableResult func preserveTabs(_ tabs: [Tab], selectedTab: Tab?) -> Success {
+    /// Async write of the tab state. In most cases, code doesn't care about performing an operation
+    /// after this completes. Deferred completion is called always, regardless of Data.write return value.
+    /// Write failures (i.e. due to read locks) are considered inconsequential, as preserveTabs will be called frequently.
+    /// - Parameters:
+    ///   - tabs: The tabs to preserve
+    ///   - selectedTab: One of the saved tabs will be saved as the selected tab.
+    ///   - writeCompletion: Used to know the write operation has completed - Used in unit tests
+    /// - Returns: <#description#>
+    @discardableResult func preserveTabs(_ tabs: [Tab], selectedTab: Tab?, writeCompletion: (() -> Void)? = nil) -> Success {
         assert(Thread.isMainThread)
-        print("preserve tabs!, existing tabs: \(tabs.count)")
+        log.debug("preserve tabs!, existing tabs: \(tabs.count)")
```

Comment:
@lmarceau so this log will add it to our browser logger
I am wondering if its not a good idea to add it to our log file 🤔

Reply:
Ahhh @nbhasin2. Do we have a better way then to log things than using `print`?

Reply:
Discussed with @nbhasin2 and the log was removed